contents/articles/2016-10_ito's_pupil_calibration_and_mapping/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Yuta Itoh's Pupil Calibration and Mapping (Windows)
+date: Mon Oct 17 2016 12:37:11 GMT+0700 (ICT)
+author: Pupil Dev Team
+subtitle: "Yuta Itoh's Pupil calibration and mapping software aims to provide an easy-to-compile C++ implementation of a 3D eye-tracking method..."
+featured_img_thumb: "../../../../media/images/blog/thumb/itoh_pupil.png"
+---
+
+Yuta Itoh's Pupil calibration and mapping software aims to provide an easy-to-compile C++ implementation of a 3D eye-tracking method.
+
+The software is designed for an egocentric vision eye-tracking headset with a world camera and IR filter eye camera(s). With the software, users can obtain 2D and 3D eye data.
```

Comment:
Their software is compatible with Pupil hardware and enables one to obtain both 2d and 3d information:

+ 3d Gaze normal vector
+ 3d eyeball center w.r.t the camera
+ Pupil diameter in mm
+ 2d pupil ellipse size in pixels
+ 2d pupil ellipse angle in degrees
+ Coming soon - glint positions